test/commands/AdventofcodeCommandTest.ts

```diff
@@ -76,7 +76,7 @@ describe("Adventofcode Command", () => {
 			sandbox.stub(command, "getYear").returns(2019);
 
 			await command.run(message, ["2021"]);
-			const embed = messageMock.getCall(0).firstArg.embed;
+			const embed = messageMock.getCall(0).firstArg.embeds[0];
```

Comment:
If you wanted you could change all of these to:
```suggestion
			const [embed] = messageMock.getCall(0).firstArg.embeds;
```

Not particularly bothered though.

Reply:
would that not give you an array of embeds? instead of just one, like we have it now?

Reply:
those are a lot of things that need to be changed, rather not. Maybe later